### PR TITLE
Fixing roc_lib test fails under valgrind

### PR DESCRIPTION
**Pull request by alexandremgo**
Fix issue #223 

I've relaxed the test requirements to have a certain percentage (for now 90%) of identical middle stream samples.

Should it raises a panic if a sample is not identical and is different from zero ?

### CODE REVIEW

**Comment by gavv** [comment]
The code looks good!

> Should it raises a panic if a sample is not identical and is different from zero ?

Yep. It would be a more correct check.

**Comment by gavv** [comment]
Also, I've done some testing and opened a new issue: #277.

File: src/tests/roc_lib/test_sender_receiver.cpp [hunk 1]
```diff
@@ -212,29 +215,40 @@ class Receiver {
...
                         roc_panic("%s", sbuff);
-                    } else {
-                        sample_num++;
                     }
                 }
             }
         }
+
+        double identical = (double)middle_stream_identical / middle_stream_total;          
+        if(identical < .9) {
```
Comment (dshil) [inline src/tests/roc_lib/test_sender_receiver.cpp]: @alexandremgo could you please run `scons -Q fmt` to use the existing formatting style?

Comment (alexandremgo) [reply src/tests/roc_lib/test_sender_receiver.cpp]: yes sorry

**Comment by gavv** [comment]
Thanks!

**Pull request merged**